Documentation/subcommands/run.md

```diff
@@ -342,11 +342,14 @@ For more details see the [hacking documentation](../hacking.md).
 
 | Flag | Default | Options | Description |
 | --- | --- | --- | --- |
+| `--cap-remove` | none | capability to remove (example: '--cap-remove=CAP\_SYS\_CHROOT,CAP\_MKNOD') | Capabilities to remove from the process's capabilities bounding set, all others from the default set will be included |
```

Comment:
I thought that it was required by the Markdown syntax. But I have not checked actually.